locust/main.py

```diff
@@ -553,7 +561,7 @@ def sig_term_handler():
         logger.info("Starting Locust %s" % version)
         main_greenlet.join()
         code = 0
-        if len(runners.locust_runner.errors):
+        if len(runners.locust_runner.errors) and options.exit_code_on_fail:
```

Comment:
This may be confusing as running locust with `--exit-code-on-fail=2` will actually exit 1 right (it'll still pass this check)?

Reply:
Should the option be an int, and post what ever exit code is specified on error?

Default to 1 (current behavior), but allow the user to select which exit code to post on errors.

Reply:
I'm actually not sure. That seems to make sense though - you could give it `0` then for the situation you want here.